### PR TITLE
Localize FXIOS-5651 [v112] Add Settings > Notification page strings

### PR DESCRIPTION
**Pull request by adudenamedruby**
This is the strings for #13461.
Doing it separately as we are string freezing today. :)

### CODE REVIEW

**Comment by adudenamedruby** [comment]
@razvanlitianu Heya. So I made a separate PR for your strings so we can get them in today. As such, you'll have to rebase your PR against main, and you'll probably have some conflicts because, you'll notice that I made some changes:
- the key is now `"Notifications.<title>"` vs `"Settings.Notifications.<title>"`. This is because...
- ...I've added `"Settings"` as the `tableName`
- I've removed hardcoded references to the name Firefox. We want to use string substitution for this. So you'll have to update your code in the relevant places. Let me know if I can help with that.

**Comment by adudenamedruby** [comment]
Only adding strings, so BR is not needed.

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1071,6 +1071,69 @@ extension String {
...
+                "Notifications.TipsAndFeaturesNotificationsTitle",
+                tableName: "Settings",
+                value: "Tips and Features",
+                comment: "This is the title for the setting that toggles Tips and Features feature in the settings menu under the Notifications section."
+            )
+            public static let TipsAndFeaturesNotificationsStatus = MZLocalizedString(
+                "Notifications.TipsAndFeaturesNotificationsStatus",
+                tableName: "Settings",
+                value: "Learn about useful features and how to get the most out of %@.",
+                comment: "This is the description for the setting that toggles Tips and Features feature in the settings menu under the Notifications section."
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1121]: Should we describe what goes into the placeholder in the comment?

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -1071,6 +1071,69 @@ extension String {
...
+            )
+            public static let TurnOnNotificationsTitle = MZLocalizedString(
+                "Notifications.TurnOnNotificationsTitle",
+                tableName: "Settings",
+                value: "Turn on Notifications",
+                comment: "This is the title informating the user needs to turn on notifications in iOS Settings."
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1127]: `informating` to `informing` ?

File: Client/Frontend/Strings.swift [hunk 3]
```diff
@@ -1071,6 +1071,69 @@ extension String {
...
+            )
+            public static let TurnOnNotificationsMessage = MZLocalizedString(
+                "Notifications.TurnOnNotificationsTitle",
+                tableName: "Settings",
+                value: "Go to your device Settings to turn on notifications in %@",
+                comment: "This is the title informating the user needs to turn on notifications in iOS Settings."
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1133]: Same thing here as well - `informing`. And also the same regarding describing the placeholder.

**Review by nishant2718 — APPROVED** [review]

**Pull request merged**